app/scripts/modules/titus/src/serverGroup/configure/serverGroupConfiguration.service.ts

```diff
@@ -312,8 +312,8 @@ export class TitusServerGroupConfigurationService {
       .map('regions')
       .flattenDeep()
       .filter({ name: this.getRegion(command) })
-      .map<IAmazonLoadBalancer>('loadBalancers')
-      .flattenDeep<IAmazonLoadBalancer>()
+      .map<IAmazonLoadBalancer>((region) => region.loadBalancers)
+      .flattenDeep()
```

Comment:
this looks a bit out of place for this PR, can you explain why this change is here?

Reply:
This is fixing a typescript failure. Weirdly, it isn't happening when running `tsc` from `deck`. I didn't spend much time on debugging the cause for the difference though.